dlc/src/main/scala/org/bitcoins/dlc/BinaryOutcomeDLCWithSelf.scala

```diff
@@ -187,46 +186,38 @@ case class BinaryOutcomeDLCWithSelf(
                                Vector(fundingLocalPubKey, fundingRemotePubKey))
   }
 
+  private val estimatedCETFee: CurrencyUnit = Satoshis(190 * feeRate.toLong)
```

Comment:
So i'm guessing `190` and `122` are estimated transaction sizes? Can you use constants for these with an appropriate scaladoc?